tox.ini

```diff
@@ -30,7 +30,8 @@ deps =
     coverage
     diff_cover
 setenv =
-    cov: COVERAGE_PROCESS_START=.coveragerc
+    cov: COVERAGE_RCFILE=coverage.cfg
+    cov: COVERAGE_PROCESS_START=coverage.cfg
```

Comment:
`.coveragerc` has been renamed to `coverage.cfg`

Reason is explained in a59fd413